climada/entity/exposures/test/test_nightlight.py

```diff
@@ -22,9 +22,10 @@
 import numpy as np
 
 from climada.entity.exposures.litpop import nightlight
-from climada.util.constants import SYSTEM_DIR
+from climada.util.constants import (SYSTEM_DIR, CONFIG)
 
 BM_FILENAMES = nightlight.BM_FILENAMES
+DATA_DIR = CONFIG.exposures.test_data.dir()
 
```

Comment:
@emanuel-schmid  for the `DATA_DIR` ? Sorry, it was a remaining of the old test that i tried, it can be delated. Regarding the `CONFIG`: since  `CONFIG`  is imported from the same file as `SYSTEM_DIR` I thought it was more  clean to import it with brackets.